### PR TITLE
Add task descriptions

### PR DESCRIPTION
**Pull request by alsco77**

### CODE REVIEW

File: dapp-developer/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# DApp Challenge
+
+So you want to work on interfaces with mStable?
+
+If you're a seasoned coder, you should have no problems with the task, but don't
+worry if you're new to the game either, we just want to see is how you solve
+problems and innovate while interacting with Ethereum.
+
+We really like to see clean, innovative interfaces and well written, modular and strongly typed code code.
```
Comment (JamesLefrere) [inline dapp-developer/README.md:9]: typo

File: dapp-developer/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Choose from existing protocols (e.g. `mStable`, `Compound`, `Aave`) as the base layer for the savings mechanism.
+
+The product should consist of:
+
+- Frontend to allow Mary to easily deposit her funds and view her accruing interest
+- Optional Smart Contract(s) to add additional functionality
+
+## Suggestions
+
+- Put a dash of colour on top with an elegant CSS library
```
Comment (alsco77) [inline dapp-developer/README.md:28]: Tweak this

File: dapp-developer/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# DApp Challenge
+
+So you want to work on interfaces with mStable?
+
+If you're a seasoned coder, you should have no problems with the task, but don't
```
Comment (JamesLefrere) [inline dapp-developer/README.md:5]: Missing the info here from the other one (e.g. what it's like to work at mStable)

File: protocol-developer/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+The ideal solution would have the following components:
+
+- Smart Contract(s)
+- Rationale for the chosen mechanism and a brief description of how it works
+  - Test output or some method of interfacing with the staking contracts would be useful
+
+## Suggestions
+
+- Test and
```
Comment (alsco77) [inline protocol-developer/README.md:31]: ? and what

File: protocol-developer/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# Protocol Developer
+
+So you want to design protocol improvements and incentive mechanisms on mStable?
+Mechanisms are hard to get right, particularly in the fast moving DeFi industry with overlapping incentives.
+
+We want to see how you solve problems and innovate while interacting with Ethereum.
+
+**Want a real world task**? Feel free to submit an upgrade or improvement to our existing protocol [mStable-contracts repo](https://github.com/mstable/mstable-contracts) instead.
```
Comment (alsco77) [inline protocol-developer/README.md:8]: or suggest with pseudocode

File: protocol-developer/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+# Task 1
+
+Design, implement, test and rationalise a solution to the following scenario.
+
+A University in Ukraine is hoping to incentivise collaboration in its community by rewarding students for starting clubs and societies. They will give all students 1000 `$COLAB` tokens each for the year. These tokens can be used to register interest and participate in activies in different student led societies, and the University will pay out
+grants to the organisations at the end of the year based on their `$COLAB` staking.
```
Comment (JamesLefrere) [inline protocol-developer/README.md:15]: Make it clear this is staked amount / time

File: protocol-developer/README.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+## Suggestions
+
+- Test and
+- Use Truffle or Buidler for creating and deploying the smart contract
+- Use Ganache-cli or Ropsten as the test blockchain
+
+### Bonus points
+
+- Build out or rationalise a weighted voting system - how much weight would each `$GAME` token have in the decisions being made to the organisation? Would they be locked up?
```
Comment (JamesLefrere) [inline protocol-developer/README.md:37]: Might be too prescriptive - it's rather a governance mechanism, which may or may not use voting

File: protocol-developer/README.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+- Gamification
+- Innovation: Show us what you can really do
```
Comment (JamesLefrere) [inline protocol-developer/README.md:39]: Another point: Vulnerabilities (how will the system be prevented from being gamed as much as possible?)

**Pull request merged**